modules/ROOT/pages/functions/temporal/format.adoc

```diff
@@ -0,0 +1,332 @@
+:description: Cypher provides a function for creating dynamically formatted string representations of temporal instance and duration types.
+:table-caption!:
+
+[[query-functions-temporal-format]]
+= Temporal functions - format
+
+[[query-functions-temporal-format-function]]
+== format()
+
+The format function creates dynamically formatted string representations of temporal instance and duration types.
+The output format can be customized via the `pattern` parameter.
+Creating a pattern follows the rules for the link:https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html[Java DateTimeFormatter].
```

Comment:
This links to Java 8, but we use 21:
https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/time/format/DateTimeFormatter.html